CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java

```diff
@@ -328,8 +334,22 @@ public void updateDisplayedAttributes() {
 
     private Map<String, Attribute> createDisplayedAttributes(final Map<String, Attribute> autoAddedAttributes, final Map<String, Attribute> manuallyAddedAttributes) {
         Map<String, Attribute> displayedAttributes = new HashMap<>();
-        displayedAttributes.putAll(manuallyAddedAttributes);
-        displayedAttributes.putAll(autoAddedAttributes);
+
+        if (filterAttributes != null &&  filterAttributes.length() > 0) {
```

Comment:
I think we have a utility method to check for both bull and blank - @arcturus2?

Reply:
isEmpty?

Reply:
StringUtils.isBlank(...) and StringUtils.isNotBlank(...)

Reply:
Sorry for the delay @cygnus-x-1 in response, @antares1470 thanks for chipping in!

Reply:
I can find only one instance of 'isBlank' being called (at , 
CoreUtilities\src\au\gov\asd\tac\constellation\utilities\temporal\TemporalFormatting.java line 497ish), no isNotBlank and about 515 isEmpty (515 empties? We are a territory but this is not NT).

Reply:
StringUtils.isBlank(str) is equivalent to str == null || str.IsEmpty().
Similarly, StringUtils.isNotBlank(str) is equivalent to str !=null && !str.isEmpty()

We probably haven't gotten around to fixing all instances yet but this is the way we'd prefer to test for emptiness (unless of course you know it can't be null in which case just use isEmpty)